content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -1198,6 +1170,181 @@ even if it is allowed by RBAC configuration.
 [User impersonation](/docs/reference/access-authn-authz/user-impersonation/) provides
 a method that a user can act as another user through impersonation headers
 
+## Authentication configuration {#api-server-authn-config}
+
+You can configure Kubernetes authentication either using
+[command line arguments](#api-server-authn-config-cli), or using a
+[configuration file](#api-server-authn-config-file).
+
+Typically, you use a mix of these approaches.
+
+### Configuration via command line arguments {#api-server-authn-config-cli}
+
+You can use the following command line arguments to configure how your cluster's control plane authenticates clients.
+
+The [command line reference](/docs/reference/command-line-tools-reference/kube-apiserver/) for the API server
+describes all of the relevant command line arguments in more detail.
+
+#### Anonymous authentication configuration {#api-server-authn-config-cli-anonymous}
+
+`--anonymous-auth`
+: Controls whether clients who have not authenticated can make request via the API server's secure port. Anonymous requests have a username of `system:anonymous`, and a group name of `system:unauthenticated`. Also see [anonymous requests](#anonymous-requests).
+
+#### Bootstrap token configuration {#api-server-authn-config-cli-bootstrap}
+
+`--enable-bootstrap-token-auth`
+: When this flag is set, you can use [bootstrap tokens](#bootstrap-tokens) to authenticate.
+
+#### Certificate authentication configuration {#api-server-authn-config-cli-x-509}
+
+`--client-ca-file`
+: The path to the trust anchor for validating client identity, when clients use X.509 certificate authentication.
```

Comment:
```suggestion
: The path to the trust anchor(s) for validating client identity, when clients use X.509 certificate authentication.
```

Also, we could note that the contents of this file can be changed while the apiserver is running and is re-read. Modifications to this file should be done in an atomic way (writing to a peer temporary file, then renaming the temporary file to replace this file, etc).